### DISCUSSION TITLE
使用Paddleslim量化PPOCRv4 det模型时报错，是因为配置文件Backbone是PPLCNetNew的问题吗？

### DISCUSSION THREAD

**Discussion by hhxdestiny** [PaddlePaddle/PaddleOCR#12816] [Q&A]
- 系统环境/System Environment：Linux
- 版本号/Version：Ubuntu22.04
- Paddle：paddlepaddle-gpu 2.5.1.post117
- PaddleOCR：release/2.7 
- 问题相关组件/Related components：PPOCRv4量化
- 运行指令/Command Code：
```bash
python deploy/slim/quantization/quant.py -c configs/det/ch_PP-OCRv4/ch_PP-OCRv4_det_cml.yml -o Global.pretrained_model='./models/ch_PP-OCRv4_det_train/best_accuracy' Global.save_model_dir=./models/ch_PP-OCRv4_det_train/quant/
```
- 完整报错/Complete Error Message：
```bash
Traceback (most recent call last):
  File "/home/rh/Workspace/ForensicsProducts/Detect/paddle/PaddleOCR/deploy/slim/quantization/quant.py", line 203, in <module>
    main(config, device, logger, vdl_writer)
  File "/home/rh/Workspace/ForensicsProducts/Detect/paddle/PaddleOCR/deploy/slim/quantization/quant.py", line 158, in main
    model = build_model(config['Architecture'])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rh/Workspace/ForensicsProducts/Detect/paddle/PaddleOCR/ppocr/modeling/architectures/__init__.py", line 34, in build_model
    arch = getattr(mod, name)(config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rh/Workspace/ForensicsProducts/Detect/paddle/PaddleOCR/ppocr/modeling/architectures/distillation_model.py", line 47, in __init__
    model = BaseModel(model_config)
            ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rh/Workspace/ForensicsProducts/Detect/paddle/PaddleOCR/ppocr/modeling/architectures/base_model.py", line 55, in __init__
    self.backbone = build_backbone(config["Backbone"], model_type)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rh/Workspace/ForensicsProducts/Detect/paddle/PaddleOCR/ppocr/modeling/backbones/__init__.py", line 77, in build_backbone
    assert module_name in support_dict, Exception(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: when model typs is det, backbone only support ['MobileNetV3', 'ResNet', 'ResNet_vd', 'ResNet_SAST', 'PPLCNet', 'PPLCNetV3', 'PPHGNet_small']
```
我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): no

**Comment by hhxdestiny** [comment]
期待官网的PPOCRv4量化模型和paddle-lite模型

**Comment by wangchenhong** [comment]
请问一下这是怎么回事，解决了吗

**Comment by ITccyhub** [comment]
我也是这个问题   File "tools/train.py", line 227, in <module>
    main(config, device, logger, vdl_writer)
  File "tools/train.py", line 135, in main
    model = build_model(config['Architecture'])
  File "D:\PaddleOCR\PaddleOCR-release-2.7\ppocr\modeling\architectures\__init__.py", line 34, in build_model
    arch = getattr(mod, name)(config)
  File "D:\PaddleOCR\PaddleOCR-release-2.7\ppocr\modeling\architectures\distillation_model.py", line 47, in __init__
    model = BaseModel(model_config)
  File "D:\PaddleOCR\PaddleOCR-release-2.7\ppocr\modeling\architectures\base_model.py", line 55, in __init__
    self.backbone = build_backbone(config["Backbone"], model_type)
  File "D:\PaddleOCR\PaddleOCR-release-2.7\ppocr\modeling\backbones\__init__.py", line 77, in build_backbone
    assert module_name in support_dict, Exception(
AssertionError: when model typs is det, backbone only support ['MobileNetV3', 'ResNet', 'ResNet_vd', 'ResNet_SAST', 'PPLCNet', 'PPLCNetV3', 'PPHGNet_small']

**Comment by hhxdestiny** [comment]
> 请问一下这是怎么回事，解决了吗

没有解决，官方尚未支持，可能需要自己改源码

**Comment by hhxdestiny** [comment]
> 我也是这个问题 File "tools/train.py", line 227, in main(config, device, logger, vdl_writer) File "tools/train.py", line 135, in main model = build_model(config['Architecture']) File "D:\PaddleOCR\PaddleOCR-release-2.7\ppocr\modeling\architectures__init__.py", line 34, in build_model arch = getattr(mod, name)(config) File "D:\PaddleOCR\PaddleOCR-release-2.7\ppocr\modeling\architectures\distillation_model.py", line 47, in **init** model = BaseModel(model_config) File "D:\PaddleOCR\PaddleOCR-release-2.7\ppocr\modeling\architectures\base_model.py", line 55, in **init** self.backbone = build_backbone(config["Backbone"], model_type) File "D:\PaddleOCR\PaddleOCR-release-2.7\ppocr\modeling\backbones__init__.py", line 77, in build_backbone assert module_name in support_dict, Exception( AssertionError: when model typs is det, backbone only support ['MobileNetV3', 'ResNet', 'ResNet_vd', 'ResNet_SAST', 'PPLCNet', 'PPLCNetV3', 'PPHGNet_small']

选了v4的配置？

**Comment by Marbyun** [comment]
hi is this using ppocrv4? where i can use the model? sorry i'm new using this one.

and want to know what's model use in the demo page? because i just use default code like in the quickstart but get different result from the same image.

**Comment by hhxdestiny** [comment]
> hi is this using ppocrv4? where i can use the model? sorry i'm new using this one.
> 
> and want to know what's model use in the demo page? because i just use default code like in the quickstart but get different result from the same image.

https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/README_en.md#%EF%B8%8F-pp-ocr-series-model-listupdate-on-september-8th

**Comment by Marbyun** [comment]
> > hi is this using ppocrv4? where i can use the model? sorry i'm new using this one.
> > and want to know what's model use in the demo page? because i just use default code like in the quickstart but get different result from the same image.
> 
> https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/README_en.md#%EF%B8%8F-pp-ocr-series-model-listupdate-on-september-8th

hi may i know how to get models in this link

https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/applications/README_en.md

i want to use the universal card recognition model. thanks
